react/components/EditorContainer/Sidebar/ConfigurationList/index.tsx

```diff
@@ -339,8 +348,16 @@ class ConfigurationList extends React.Component<Props, State> {
   }
 
   private handleConfigurationDiscard = () => {
-    this.props.formMeta.setWasModified(false, () => {
+    const { editor, iframeRuntime } = this.props
+
+    this.props.formMeta.setWasModified(false, async () => {
```

Comment:
The second argument of `setWasModified` could be replaced with a call to `handleContentBack`.